graphql/multiplication-connect/createUserData.ts

```diff
@@ -0,0 +1,9 @@
+import { gql } from "@apollo/client";
+
+export const CREATE_USER_MC_DATA = gql`
+  mutation createUser($id: String = "wHZGkjB7CFf0cMrcGgI1pIHfke02") {
+    insert_multiplicationConnectData(objects: { id: $id }) {
+      affected_rows
+    }
```

Comment:
I see you have if/then logic that makes sure the data is empty and then inserts.  Just a thought...You may want to be double sure and add a constraint in the query where it will only insert if the primary key id doesn't already exist.

here's an example:

mutation upsertLongestStreakLevel($userId: String = "", $currentLevel: Int = 1) {
    insert_longestStreakUserData_one(object: {userId: $userId, currentLevel: $currentLevel}, on_conflict: {constraint: longestStreakUserData_pkey, update_columns: currentLevel, where: {userId: {_eq: $userId}}}) {
      currentLevel
      userId
    }
  }